zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -678,6 +925,58 @@ components:
         endDate:
           type: string
           format: date-time
+    CamundaUserDto:
+      type: "object"
+      properties:
+        id:
+          type: "integer"
+          format: "int64"
+        username:
+          type: "string"
+        name:
+          type: "string"
+        email:
+          type: "string"
+        enabled:
+          type: "boolean"
+    CamundaUserWithPasswordDto:
+      allOf:
+        - $ref: '#/components/schemas/CamundaUserDto'
+        - type: "object"
+          properties:
+            password:
+              type: "string"
+    FilterDto:
+      type: "object"
+      properties: { }
+    SortDto:
+      type: "object"
+      properties: { }
+    PageDto:
+      type: "object"
+      properties: { }
+    SearchRequestDto:
+      type: "object"
+      properties:
+        filter:
+          type: "object"
+          properties: { }
+        sort:
+          type: "object"
+          properties: { }
+        page:
+          type: "object"
+          properties: { }
+    UserSearchResponseDto:
```

Comment:
🔧 Please have a look at `ProcessInstanceSearchQueryResponse` to see how to reuse the page response schema and extend it accordingly.

Reply:
Done that, thanks :)